doc/examples/example1.md

```diff
@@ -0,0 +1,119 @@
+Asymmetric Key Example {#example1}
+==================================
```

Comment:
A minor thing, but format using in the examples is by far not the "native" markdown format for GitHub (or gitlab). I would expect to see # for headers and \```c++ \``` for the code. Is it due to Doxygen limitations in rendering markdown files?

Reply:
Yes, this is because of doxygen. The titles are also supported by github, but the \code tags are doxygen specific.